### DISCUSSION TITLE
how to debug postman phpstorm?

### DISCUSSION THREAD

**Discussion by anhdung1301** [wardenenv/warden#730] [Help & Support]
### Version of Warden

0.12.0

### Operating System and Installation Method

Ubuntu

### Describe the Bug

how to debug postman phpstorm?

**Comment by bap14** [comment]
Follow the instructions here to set up PHPStorm for xdebug in a Warden project: https://docs.warden.dev/configuration/xdebug.html

In Postman you just need to add an `XDEBUG_SESSION` for your warden domain. Se the value to "PHPSTORM" and the expiration to some time in the future:

    XDEBUG_SESSION=PHPSTORM; Path=/; Expires=Sun, 08 Feb 2099 19:49:16 GMT;

Make sure your PHPStorm is listening for debug connections, then execute your request in Postman.
